usagov/vote-gov-drupal
# VOTE-909: Footer styling - email signup and identifier

## Jira ticket (required)
[VOTE-909](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-909&assignee=609161faf6c0960069bf61b3)
[VOTE-911](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-911&assignee=609161faf6c0960069bf61b3)

## Description (optional)
Styled the sign up block and identifier in the footer region

### Testing steps
1. Checkout branch feature/VOTE-909/style-footer and run lando retune command to sync
2. Compare markup and style of footer region to current Vote.gov site and confirm they look the same and use the same styles.
